Feat(content): A new set of `entering` bounty jobs (#9532)
endless-sky/endless-sky: data/human/jobs.txt

			payment 25000
			fail

mission "Bounty Hunting (Very Tiny, Local)"
	name "Wanted corsair entering <system>"

> 	description "A small pirate fighter named the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer: ```suggestion
	description "A small pirate fighter named the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```
I feel like this would read a bit better and avoid some confusion.

Reviewer 2: Maybe:
```suggestion
	description "A small pirate fighter named the <npc> has been attacking merchants in nearby star systems, and now it's about to arrive in <system>! Destroy it by <day> for payment (<payment>)."
```

Reviewer: That works for me.

Reviewer 3: Here definitely works best, imo, because it far better emphasizes current location than just referencing it.

IF we really want to include system name for -total- player clarity, it would be better to still do "here, in <system>," but that's an awkward presentation that strains in character tone.

			payment 25000
			fail

mission "Bounty Hunting (Very Tiny, Local)"
	name "Wanted corsair entering <system>"

> 	description "A small pirate fighter named the <npc> has been attacking merchants in nearby star systems, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer 4: ```suggestion
	description "A small pirate fighter named the <npc> has been attacking merchants in nearby star systems, and is reportedly entering this system. Destroy it by <day> for payment (<payment>)."
```

	repeat
	job
	deadline 3
	"apparent payment" 25000
	to offer
		or
			"combat rating" > 15
			has "ships: Interceptor"
			has "ships: Light Warship"
		"combat rating" < 80

> 		random < 30

Reviewer 4: ```suggestion
		random < 15
```

			has "ships: Interceptor"
			has "ships: Light Warship"
		"combat rating" < 200

> 		random < 35

Reviewer 4: ```suggestion
		random < 30
```

			payment 50000
			fail

mission "Bounty Hunting (Tiny, Local)"
	name "Wanted outlaw entering <system>"

> 	description "A well armed pirate fighter named the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer: ```suggestion
	description "A well armed pirate fighter named the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```

			payment 50000
			fail

mission "Bounty Hunting (Tiny, Local)"
	name "Wanted outlaw entering <system>"
	description "A well armed pirate fighter named the <npc> has been attacking merchants in nearby star systems, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."
	repeat
	job
	deadline 3
	"apparent payment" 50000
	to offer
		or
			"combat rating" > 30
			has "ships: Interceptor"
			has "ships: Light Warship"
		"combat rating" < 200

> 		random < 25

Reviewer 4: ```suggestion
		random < 20
```

			has "ships: Light Warship"
			has "ships: Medium Warship"
		"combat rating" < 350

> 		random < 30

Reviewer 4: ```suggestion
		random < 25
```

			payment 95000
			fail

mission "Bounty Hunting (Very Small, Local)"
	name "Wanted criminal entering <system>"

> 	description "A light pirate warship named the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer: ```suggestion
	description "A light pirate warship named the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```

	to offer
		"combat rating" > 200
		"combat rating" < 500

> 		random < 30

Reviewer 4: ```suggestion
		random < 25
```

			payment 180000
			fail

mission "Bounty Hunting (Small, Local)"
	name "Wanted bandit entering <system>"

> 	description "A medium pirate warship named the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer: ```suggestion
	description "A medium pirate warship named the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```

	to offer
		"combat rating" > 350
		"combat rating" < 1200

> 		random < 30

Reviewer 4: ```suggestion
		random < 25
```

			payment 240000
			fail

mission "Bounty Hunting (Medium, Local)"
	name "Wanted pirate entering <system>"

> 	description "A sizeable pirate warship named the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer: ```suggestion
	description "A sizeable pirate warship named the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```

	to offer
		"combat rating" > 500
		"combat rating" < 2500

> 		random < 30

Reviewer 4: ```suggestion
		random < 25
```

			payment 300000
			fail

mission "Bounty Hunting (Big, Local)"
	name "Wanted warlord entering <system>"

> 	description "A heavy warship known as the <npc> has been attacking merchants near the <system> system, and now it's about to arrive here! Destroy it by <day> for payment (<payment>)."

Reviewer: ```suggestion
	description "A heavy warship known as the <npc> has been attacking merchants nearby, and now it's about to arrive here in the <system> system! Destroy it by <day> for payment (<payment>)."
```
